ltr_tgs.txt

```diff
@@ -48,44 +40,38 @@
 |- align= "center
 |0-17-103||Waltham||Armstrong Ambulance Chair Car||2022.10/.05 OceanNora
 |- align = "center"
-|1-01-102||Hudson||To be Identified (Plowing Nashua NH)||2022.01.07 Dave2582
+|0-01-102||Hudson||To be Identified (Plowing Nashua NH)||2022.01.07 Dave2582
 |- align="center"
 |1-10-103||Boston||To be Identified - Transport/Delivery Service in/around Waltham||2020.12.28 ecps92
 |- align="center"
-|1-12-104||||Armstrong ALS backup channel||2017.05.29 N1ZYY
+|1-12-104||Boston||Armstrong ALS backup channel||2017.05.29 N1ZYY
 |- align="center"
-|1-12-105||||Armstrong BLS backup channel||2017.05.29 N1ZYY
+|1-12-105||Boston||Armstrong BLS backup channel||2017.05.29 N1ZYY
 |- align="center"
-|1-12-106||||Armstrong BLS/May also include Chair Cars||2017.11.03 ecps92
+|1-12-106||Boston||Armstrong BLS/May also include Chair Cars||2017.11.03 ecps92
 |- align="center"
-|1-01-107||Hudson||To be Identified (Plowing Nashua NH)||2018.01.10 N1ZYY
-|- align="center"
-|1-xx-127||||To be identified (Taxi or TowCo)||2017.05.29 N1ZYY
+|0-01-107||Hudson||To be Identified (Plowing Nashua NH)||2018.01.10 N1ZYY
 |- align="center"
 |0-01-132||Waltham||To be Identified (Plowing/Construction)||2022.11.22 N1ZYY
 |- align="center"
-|1-12-140||Waltham||To be Identified (Poss Taxi) Voice Inversion||2017.09.04 ecps92
-|- align="center"
-|1-12-141||Waltham||To be Identified (Poss Taxi) Voice Inversion||2017.09.04 ecps92
-|- align="center"
```

Comment:
This is a more debatable removal; the voice inversion was quite distinctive. But they've been gone 5+ years and weren't really identified.